blueprintapi/urls.py

```diff
@@ -34,6 +35,7 @@
 urlpatterns = [
     path("", views.index, name="index"),
     path("admin/", admin.site.urls),
+    path('api-token-auth/', views.UserObtainTokenView.as_view()),
```

Comment:
I intentionally didn't put the auth token endpoint under the `/api` path, but if folks would prefer that it gets moved there, let me know.

Reply:
I only suggested this for consistency with the other endpoints that the UI calls, but the front end related code already is handling this exception to the rule, so it is fine with me.

Reply:
I doubt this is going to hurt anyone's feelings.